cert_schema/schema_validator.py

```diff
@@ -88,6 +90,13 @@ def validate_v3_alpha(certificate_json, ignore_proof=False):
             schema_json['required'].remove('proof')
         return validate_json(certificate_json, schema_json)
 
+def validate_v3_beta(certificate_json, ignore_proof=False):
+    with open(SCHEMA_FILE_V3_BETA) as schema_f:
```

Comment:
does this automatically load external files (ie: displaySchema.json)?

Reply:
yes. it opens a file and returns it as a file object.